roles/common/postgres/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install postgresql
+  apt:
+    name: postgresql
+    state: present
+    update_cache: yes
+
+- name: Install psycopg2
+  pip:
+    name: psycopg2
+    version: 2.7.3.2
+    state: present
+
+- name: Create postgres role for custom user
+  become: true
+  become_user: postgres
+  postgresql_user:
+    name: "{{ custom_user }}"
```

Comment:
It may be desired to have multiple users doing different tasks.

Suggest changing the variable name to indicate that it's to be the owner of a postgres database.

Reply:
done